flexmeasures/api/common/utils/deprecation_utils.py

```diff
@@ -1,12 +1,39 @@
 from __future__ import annotations
 
-from flask import current_app, request, Blueprint, Response, after_this_request
+from flask import abort, current_app, request, Blueprint, Response, after_this_request
 from flask_security.core import current_user
 import pandas as pd
 
 from flexmeasures.utils.time_utils import to_http_time
 
 
+def sunset_blueprint(
```

Comment:
I'm opting not to, in case we ever want to use it on non-API blueprints. It's quite generic already (just like `def deprecate_blueprint` is) and should also work on e.g. UI blueprints. We can move and revise the function should that situation arise, but for now I just like how the name reflects its potential use on any blueprint.